index.js

```diff
@@ -306,7 +309,9 @@ async function connect (options) {
       renderer.domElement.mozRequestPointerLock ||
       renderer.domElement.webkitRequestPointerLock
     document.addEventListener('mousedown', (e) => {
-      renderer.domElement.requestPointerLock()
```

Comment:
Might want to put this in a separate PR

Reply:
Maybe, however the edit is required for the game menu to function over all, so might as well add it for the chat too?

Reply:
What do you think? Keep it here or put it in a separate PR, I will need to do it for when the user is in the game menu anyway, so felt natural to include chat with it in this PR. But if you believe the chat bit should be in a separate PR I will do that.

Reply:
Let's keep it here